app/src/main/java/com/example/android/adapter/ArtistAdapter.kt

```diff
@@ -1,24 +1,32 @@
 package com.example.android.adapter
 
+import android.annotation.SuppressLint
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.ImageView
 import android.widget.TextView
 import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.resource.bitmap.CircleCrop
+import com.bumptech.glide.request.RequestOptions
 import com.example.android.R
 import com.example.android.model.Artist
-import com.example.android.model.Song
-import java.text.NumberFormat
 import java.util.*
 
-class ArtistAdapter (private val item: List<Artist>) :
+class ArtistAdapter(private var item: List<Artist>) :
```

Comment:
use diff util might be help
https://viblo.asia/p/su-dung-diffutil-trong-android-recyclerview-WAyK84GeKxX